corehq/apps/sms/templates/sms/usage_report.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'hqwebapp/base_section.html' %}
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load timezone_tags %}
+{% requirejs_main "sms/js/sms_usage_report" %}
+{% block page_content %}
+    <form method="post">
```

Comment:
we follow indentation of 2 spaces for HTML.

Reply:
Fixed